v3/bench_test.go

```diff
@@ -0,0 +1,59 @@
+package h3
```

Comment:
This file (and others in v3) are all copied from the repo's current root?

Reply:
that's right.  everything just copied here.  I don't like this solution too much.  The alternative I thought of is to manually alias everything exported in the root here.  However, this links the root module and the versioned module together, which is generally a no-no.  I added more information about what I'd like to do moving forward after this duplication to the associated issue #24 .  

I will add this to the Contribution guide and Changelog in this PR